packages/ember-routing/lib/ext/controller.js

```diff
@@ -16,6 +16,8 @@ ControllerMixin.reopen({
     `this.category` and `this.page`.
     By default, Ember coerces query parameter values using `toggleProperty`.
     This behavior may lead to unexpected results.
+    Available queryParam types: `boolean`, `number`, `array`
```

Comment:
Also `string`?

Reply:
@btecu nope, https://github.com/emberjs/ember.js/blob/6fc89cdf13124d88b8ae6adf99bb02a8c0cdf508/packages/ember-routing/lib/system/router.js#L732

Reply:
Isn't `string` the default?

Reply:
it can be `null`, `undefined`, or `string`

Reply:
in happy path - `string`